$NetBSD: patch-config_Makefile.freebsd,v 1.1 2014/12/31 16:08:51 rumko Exp $

On FreeBSD, we need -fPIC in order to link the final library.
See https://sourceforge.net/p/glew/bugs/264/

--- config/Makefile.freebsd.orig	2014-12-27 09:21:52.872201275 +0000
+++ config/Makefile.freebsd
@@ -5,7 +5,7 @@ LDFLAGS.EXTRA = -L/usr/pkg/lib
 LDFLAGS.GL = -lGL -lX11
 LDFLAGS.STATIC = -Wl,-Bstatic
 LDFLAGS.DYNAMIC = -Wl,-Bdynamic
-CFLAGS.EXTRA += -I/usr/X11R6/include
+CFLAGS.EXTRA += -I/usr/X11R6/include -fPIC
 NAME = GLEW
 WARN = -Wall -W
 POPT = -O2
